travis/test-install.sh

```diff
@@ -1,13 +1,18 @@
 set -e
 
 # Check that a source distribution can be successfully generated, and that
-# the generated source distribution can be installed
+# the generated source distribution can be installed and tested
 cabal sdist
 if SRC_TGZ="$PWD/dist/$(cabal info . | awk '{print $2;exit}').tar.gz"
 then
-  mkdir -p ../install-test
-  cd ../install-test
-  mkdir -p sandboxes/$GHCVER/${STACKAGE:-none}
-  cabal sandbox init --sandbox sandboxes/$GHCVER/${STACKAGE:-none}
-  cabal install -j2 --ghc-options='+RTS -A32m -RTS' "$SRC_TGZ"
+  if [ "$RUNSDISTTESTS" = "YES" ]; then
```

Comment:
So we can run tests successfully even when we fetch purescript from hackage, e.g. stackage test runner does that for us nightly (or would do soon)